frontend/src/metabase/visualizations/visualizations/Sankey.jsx

```diff
@@ -0,0 +1,234 @@
+/* eslint-disable react/prop-types */
+import React, { Component } from "react";
+import ReactDOM from "react-dom";
+import { t } from "ttag";
+import d3 from "d3";
+import cx from "classnames";
+
+import _ from "underscore";
+
+import { isNumeric } from "metabase-lib/types/utils/isa";
+
+import { sankey, sankeyLinkHorizontal } from "d3-sankey";
+import chroma from "chroma-js";
+
+import {
+  SANKEY_DATA_SETTINGS,
+} from "../lib/settings/graph";
+
+const MAX_WIDTH = 500;
+const PADDING_BOTTOM = 10;
+
+export default class Sankey extends Component {
```

Comment:
For new components we use functional components instead of class compoenets.

Reply:
Is there an example one that I can try to model after?

Reply:
Here’s one https://github.com/metabase/metabase/blob/70f921b0b1de85d62c3ad8742454c16fba6589b4/frontend/src/metabase/visualizations/visualizations/RowChart/RowChart.tsx#L103)